### PR TITLE
[12.0][FIX] base_import_module_group: fix group external id and add test

### PR DESCRIPTION
**Pull request by yankinmax**

### CODE REVIEW

File: base_import_module_group/tests/test_security_access.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                "email": "test_user@example.com",
+                "groups_id": [cls.env.ref("base.group_user").id,],  # noqa: E231
+            }
+        )
+        cls.test_admin = cls.env["res.users"].create(
+            {
+                "name": "Admin",
+                "login": "test_admin",
+                "email": "test_admin@example.com",
+                "groups_id": [cls.env.ref("base.group_system").id,],  # noqa: E231
```
Comment (grindtildeath) [inline base_import_module_group/tests/test_security_access.py]:
```suggestion
                "groups_id": [cls.env.ref("base_import_module_group.group_module_import").id,],  # noqa: E231
```

**Review by grindtildeath — APPROVED** [review]
LG, thx for the fix and the test :+1: 

Just a small remark on the group being used in the test, pre approving anyway :wink:

**Comment by thomaspaulb** [comment]
@yankinmax Tests are failing on "Cannot set admin user as a readonly user"

**Comment by yankinmax** [comment]
@thomaspaulb thanks for your response on that.
Unfortunately, I don't have time to set up v12 to check this PR. I'm very sorry about this.
If you need this to be merged you can try to fix by yourself. I propose to open new PR and ping me on that.

**Pull request closed without merging**